pints/_mcmc/_adaptive_covariance.py

```diff
@@ -165,6 +166,9 @@ def tell(self, fx):
         # Ensure fx is a float
         fx = float(fx)
 
+        # Increase iteration count
+        self._iterations += 1
```

Comment:
Again, we don't use these here, so why are they this class's responsibility?

Reply:
Not sure where else to put these -- they are used by the controller, so I put them in the highest class I could?

Reply:
No they're not :D
The controller should not, and as far as I can see does not, access other objects private properties.
The iteration count inside the original AdaptiveCovarianceMCMC was maintained purely to calculate an acceptance rate for user output

Reply:
Ok. The way I have it at the moment, all adaptive methods require an iteration count so that we can calculate an acceptance rate (``self._acceptance``) which is logged by the controller. To me, to avoid having to define an iteration counter in each method, it seems sensible to have this in the base class. What do you think?